docs/data/real_property.md

```diff
@@ -0,0 +1,16 @@
+---
+---
+layout: http://dcpres.urban.org/dcp/
+https://otr.cfo.dc.gov/service/real-property-taxpayers
+tablename: real_property
+title: Office of Tax and Revenue's (OTR) Real Property Tax Database March 2017
+---
+<!--No need to put a header; the title in the front matter (above) will be used as a header-->
+
+The Office of Tax and Revenue's (OTR) real property tax database provides online access to real property information including property value, assessment roll, and other information for more than 180,000 parcels.  
+
+The data source was downloaded from the [Urban Institute Preservation Catalog](http://dcpres.urban.org/dcp/), who mapped the data from [DC Office of Tax and Revenue's database](https://otr.cfo.dc.gov/service/) to the project database.  The challenge of this data is that this is a manual upload from March 2017, and has not been set up to be updated regularly.  
```

Comment:
Can you clarify if the table we have from PresCat is a manual upload, or if it is the file that Urban used in their process that was manual? We plan to get monthly versions of the PresCat once we're done w/ v2